src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
Ok , I understand you concern now. You are talking about using default [`CacheManager`](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/cache/concurrent/ConcurrentMapCacheManager.html) provided by Spring Boot OotB, right?